testGeogrid_ISCE.py

```diff
@@ -279,22 +273,46 @@ def runGeogrid(info, info1, dem, dhdx, dhdy, vx, vy, srx, sry, csminx, csminy, c
     obj.winssmname = "window_stable_surface_mask.tif"
     obj.winro2vxname = "window_rdr_off2vel_x_vec.tif"
     obj.winro2vyname = "window_rdr_off2vel_y_vec.tif"
-    
+
     obj.getIncidenceAngle()
     obj.geogrid()
 
-
-
-def runGeogridOptical(info, info1, dem, dhdx, dhdy, vx, vy, srx, sry, csminx, csminy, csmaxx, csmaxy, ssm, urlflag):
+    run_info = {
+        'chipsizex0': obj.chipsizex0,
+        'vxname': vx,
+        'vyname': vy,
+        'sxname': srx,
+        'syname': sry,
+        'maskname': kwargs.get('sp'),
+        'xoff': None,  # FIXME: Get from C object (is calculated) or another source
+        'yoff': None,  # FIXME: Get from C object (is calculated) or another source
+        'xcount': None,  # FIXME: Get from C object (is calculated) or another source
+        'ycount': None,  # FIXME: Get from C object (is calculated) or another source
+        'dt': obj.repeatTime,
+        'epsg': kwargs.get('epsg'),
+        'XPixelSize': None,  # FIXME: Get from C object (is calculated) or another source
+        'YPixelSize': None,  # FIXME: Get from C object (is calculated) or another source
+        'pixsizex': None,  # FIXME: Get from C object (is calculated) or another source
+        'rangePixelSize': None,  # FIXME: Get from C object (is calculated) or another source
+        'azimuthPixelSize': None,  # FIXME: Get from C object (is calculated) or another source
+    }
+
```

Comment:
I haven't done the work to figure out how to get these parameters in and out of the C++ object, or get them from another location...